doc/xml/reference.xml

```diff
@@ -1584,15 +1585,15 @@
                     <option id="target" name="Target">
                         <summary>Recovery target.</summary>
 
-                        <text>Defines the recovery target when <br-option>--type</br-option> is <id>name</id>, <id>xid</id>, or <id>time</id>.</text>
+                        <text>Defines the recovery target when <br-option>--type</br-option> is <id>lsn</id>, <id>name</id>, <id>xid</id>, or <id>time</id>.</text>
```

Comment:
We've been meaning to update this, so this is a good time. Can you also add the following sentence to this text:
`If the target is prior to the latest backup and <br-option>--type</br-option> is not <id>time</id>, then use the <br-option>--set</br-option> option to specify the backup set.`

Reply:
I committed this change separately in a6691c6f615f9 and merged it into this branch.